doc/research/Grundrecherche_im_Planing.md

```diff
@@ -0,0 +1,261 @@
+# Grundrecherche im Planing
+
+## PAF 2021-1
+
+### Vehicle Controller
+
+Ausgabe dieser Komponente ist ein "DrivingSignal" bestehend aus einer Sollgeschwindigkeit und einem Lenkwinkel. Außerdem ist der Controller dazu in der Lage Kurven zu detektieren und eine Geschwindigkeit zu errechnen mit der diese durchführbar ist.

```

Comment:
Steht in paf 21 unter dem Bereich planing, gehört aber glaube ich eher zu acting in unserem Projekt